qutebrowser/javascript/webengine_caret.js

```diff
@@ -0,0 +1,1104 @@
+/**
+ * Copyright 2016-2017 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
```

Comment:
Remove the `2016-` :wink:

Reply:
I think we can go further and change 2017 for 2018 already. 🎅

Reply:
Or 2017-2018, however I'll update that across the codebase anyways :wink: